packages/binding-builder-cli/scripts/build.js

```diff
@@ -0,0 +1,414 @@
+const path = require("node:path");
+const { readFileSync, writeFileSync } = require("node:fs");
+
+const debug = require("debug")("rspack-builder:build");
+
+const OPTIONS_CONFIG = {
```

Comment:
is this duplicate of https://github.com/napi-rs/napi-rs/blob/main/cli/src/def/build.ts#L5 or we add custom logic here?